src/services/apis/bbga.js

```diff
@@ -23,11 +23,11 @@ function getUrl(endpoint) {
  */
 export async function getAllMeta() {
   async function getData() {
-    const url = getUrl('/meta/')
-    const data = await readPaginatedData(url)
+    const url = getUrlv1('/indicatoren_definities/?page_size=1000')
+    const data = await readData(url)
```

Comment:
By using readData instead of readPaginatedData, only the first page is fetched.

For example:

Navigate to`http://localhost:8080/#/dashboard?gebied=DX04&wijk=E38&buurt=E38c&thema=Economie%20en%20cultuur` -> Verschillen binnen het gebied -> Select "Leegstand kantoorruimte" 

This initiates a request with more than 1000 results, but only the first page (with 1000 results) is fetched. Does this mean that the rendered data is incomplete?

Reply:
the readPaginatedData has page_size=1000 inside: so it does not paginate at all. that is why I use the readData. also the results are now embedded in data._embedded.indicatoren_definities. that is not compliant result set.
<img width="742" alt="Screenshot 2021-06-10 at 10 35 32" src="https://user-images.githubusercontent.com/4640157/121493466-283ae780-c9d8-11eb-9e05-5b64c770903b.png">

Reply:
Indeed `pageSize` is 1000 (constant), but the `page` is being iterated. With `readPaginatedData`, a set of 1500 results in two requests (one for the first page - 1000 results and one for the remaining 500).